src/js/collections/maps/MapAssets.js

```diff
@@ -98,6 +98,8 @@ define([
         }
       },
 
+      modelId: attrs => attrs.cid,
```

Comment:
Oh no, what a frustrating sounding issue!! It seems like having "id" as a model attribute in MapAsset is a conflict with backbone. This attribute is intended to be used to specify the package ID for an associated dataset on a dataone repo (like the Arctic Data Center). I think we could rename this `pid` (what the IDs are usually called on dataone), or maybe `datapackage_id`. This will mean updating the PDG and DRP portal xml, but I think it's better we do this now while cesium isn't in wide use and we have control over the portal document content.